C++/include/algorithm/dynamic_programming/longest_subsequence.hpp

```diff
@@ -0,0 +1,73 @@
+#ifndef LONGEST_SUBSEQUENCE
+#define LONGEST_SUBSEQUENCE
+
+/*
+Made by Cole Severson
+<Longest Subsequence>
+---------------------
+This algorithm finds either the longest increasing (or decreasing depending on the order varialbe) subsequence in a sequence of numbers
+It follows the principals of dynamic programming
+
+Time complexity
+----------------
+O(n logn) where n is the number of elements in the initial list
+
+Space complexity
+---------------
+O(n)
+
+*/
+
+
+#include <iostream>
+#include <vector>
+
+
+
+
```

Comment:
Add function comments following this pattern: https://github.com/faheel/Algos/blob/reorganise/C%2B%2B/include/algorithm/backtracking/n_queens.hpp